render/detailed_node.go

```diff
@@ -313,6 +313,18 @@ func containerOriginTable(nmd report.NodeMetadata, addHostTag bool) (Table, bool
 		rows = append(rows, Row{Key: "IP Address", ValueMajor: ip, ValueMinor: ""})
 	}
 
+	// Add labels in alphabetical order
+	labels := docker.ExtractContainerLabels(nmd)
+	labelKeys := make([]string, 0, len(labels))
+	for k := range labels {
+		labelKeys = append(labelKeys, k)
+	}
```

Comment:
Oh! I'm so sorry, I misread `labels` type. You're totally right. Disregard my comment.